axelrod/tests/strategies/test_backstabber.py

```diff
@@ -24,58 +24,100 @@ def test_strategy(self):
+
+    def _defects_on_last_two_rounds_by_match_len(self):
+        actions = [(C, C)] * 198 + [(D, C), (D, C)]
+        self.versus_test(axelrod.Cooperator(), expected_actions=actions, match_attributes={"length": 200})
+
+        actions = [(C, C)] * 10 + [(D, C), (D, C)]
+        self.versus_test(axelrod.Cooperator(), expected_actions=actions, match_attributes={"length": 12})
 
-        # Defects on rounds 199, and 200 no matter what
-        self.responses_test([C, D, D], [C] * 197, [C] * 197, length=200)
         # Test that exceeds tournament length
-        self.responses_test([D, D, C], [C] * 198, [C] * 198, length=200)
+        actions = [(C, C)] * 198 + [(D, C), (D, C), (C, C), (C, C)]
+        self.versus_test(axelrod.Cooperator(), expected_actions=actions, match_attributes={"length": 200})
         # But only if the tournament is known
-        self.responses_test([C, C, C], [C] * 198, [C] * 198, length=-1)
+        actions = [(C, C)] * 202
+        self.versus_test(axelrod.Cooperator(), expected_actions=actions, match_attributes={"length": -1})
 
 
-class TestDoubleCrosser(TestPlayer):
+class TestDoubleCrosser(TestBackStabber):
 
     name = "DoubleCrosser"
     player = axelrod.DoubleCrosser
     expected_classifier = {
         'memory_depth': float('inf'),
         'stochastic': False,
-        'makes_use_of': set(['length']),
+        'makes_use_of': {'length'},
         'long_run_time': False,
         'inspects_source': False,
         'manipulates_source': False,
         'manipulates_state': False
     }
 
     def test_strategy(self):
-
         """
-        Forgives the first 3 defections but on the fourth will defect forever.
-        If the opponent did not defect in the first 6 rounds the player will
-        cooperate until the 180th round. Defects after the 198th round
-        unconditionally.
+        Forgives the first 3 defections but on the fourth
+        will defect forever. Defects on the last 2 rounds unconditionally.
+
+        If the opponent did not defect
+        in the first 6 rounds,until the 180th round, the player will only defect
+        after the opponent has defected twice in-a-row.
         """
-        self.first_play_test(C)
 
-        # Forgives three defections
-        self.responses_test([C], [C], [D], length=200)
-        self.responses_test([C], [C, C], [D, D], length=200)
-        self.responses_test([C], [C, C, C], [D, D, D], length=200)
-        self.responses_test([D], [C, C, C, C], [D, D, D, D], length=200)
-
-        # If opponent did not defect in the first six rounds, cooperate until
-        # round 180
-        self.responses_test([C] * 174, [C] * 6, [C] * 6, length=200)
-        self.responses_test([C] * 160, [C] * 12, [C] * 6 + [D] + [C] * 5,
-                            length=200)
-
-        # Defects on rounds 199, and 200 no matter what
-        self.responses_test([C, D, D], [C] * 197, [C] * 197, length=200)
+        self._when_alt_strategy_is_triggered()
```

Comment:
Could you put these as their own test methods.

So remove them from here and rename them to start with the word `test`:

```
_when_alt_stragegy_is_triggered -> test_when_alt_strategy_is_triggered
...
```

This would make the test failures easier to read (by having them as specific tests instances so to speak).